server/config/config.go

```diff
@@ -818,7 +818,7 @@ func (man Manager) addConfigs() {
 	man.addConfigInt("redis.max_open_conns", 0, "Redis maximum open connections, 0 means no limit")
 	man.addConfigDuration("redis.conn_max_lifetime", 0, "Redis maximum amount of time a connection may be reused, 0 means no limit")
 	man.addConfigDuration("redis.idle_timeout", 240*time.Second, "Redis maximum amount of time a connection may stay idle, 0 means no limit")
-	man.addConfigDuration("redis.conn_wait_timeout", 0, "Redis maximum amount of time to wait for a connection if the maximum is reached (0 for no wait, ignored in non-cluster Redis)")
+	man.addConfigDuration("redis.conn_wait_timeout", 0, "Redis maximum amount of time to wait for a connection if the maximum is reached (0 for no wait)")
```

Comment:
@rfairburn is this 0 in the customer environment?

Reply:
Correct it is currently 0 because we are effectively standalone (a single replication group with a rw and 2 readers) and don't have it explicitly set because it wouldn't have worked previously anyhow.

Reply:
@lucasmrod just to be sure this is clear - that value was ignored in non-Redis Cluster environments prior to this PR (and the customer environments were non-cluster AFAIK).